resources/prometheusrule-alerts/node-alerts.yaml

```diff
@@ -95,3 +95,11 @@ spec:
       for: 15s
       labels:
         severity: info-warning
+    - alert: NodeNotReadyState
+      annotations:
+        message: 'The Node is in a state of NotReady, investigate further'
+        runbook_url: https://github.com/ministryofjustice/cloud-platform-terraform-monitoring/blob/main/resources/prometheusrule-alerts/README.md
+      expr: (kube_node_status_condition{condition="Ready", status="true"})
```

Comment:
Should this `condition=NotReady`?

Reply:
Yes, that should be NotReady, well spotted, thanks